libcloud/compute/drivers/upcloud.py

```diff
@@ -109,10 +110,16 @@ def list_sizes(self):
         """
         List available plans
 
+        Note: Node.price will be always None, because the pricing depends,
+        where the Node is hosted. Node.extra['zones'] will contain
```

Comment:
This makes some sense, but I'll note that other drivers have handled this by requiring a NodeLocation as an argument, and setting `NodeSize.price` accordingly. Consider switching to this approach, for consistency.

Reply:
Yeah it is better that way, I think the parameter must be named ex_location. I will change this.

Reply:
On the contrary, the parameter is just `location` in the drivers I've worked with so far. At the least, both Azure ARM and Amazon EC2 use `location=None` as the argument definition.

Reply:
Ok, I'll go with the plain location. If it is approved its ok.